views/accessibility-statement.html

```diff
@@ -60,35 +53,32 @@ <h2 class="govuk-heading-m">Enforcement procedure</h2>
-      <p>This service is fully compliant with the
-        <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a>
+      <p>This service is partially compliant with the
+        <a href="https://www.w3.org/TR/WCAG22/">Web Content Accessibility Guidelines version 2.2</a>
         AA standard.</p>
 
       <h2 class="govuk-heading-m">How we tested this service</h2>
-      <p>We regularly carry out internal tests for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA using a variety of manual and automated testing tools.</p>
+      <p>We regularly carry out internal tests for compliance with the Web Content Accessibility Guidelines V2.2 level A and level AA using a variety of manual and automated testing tools.</p>
       <p>This service was audited on 7 December 2021. The audit was carried out by the <a href="https://digitalaccessibilitycentre.org/">Digital Accessibility Centre</a>. The complete end to end user journey was audited.</p>
 
       <h2 class="govuk-heading-m">What we’re doing to improve accessibility</h2>
 
       <p>To improve this service's accessibility, we:
       <ul class="govuk-list govuk-list--bullet">
         <li>work with real users who have accessibility issues to make sure that our services meet their needs</li>
-        <li>carry out internal tests for compliance with Web Content Accessibility Guidelines V2.1 level A and level AA</li>
+        <li>carry out internal tests for compliance with Web Content Accessibility Guidelines V2.2 level A and level AA</li>
         <li>work with external agencies in major UK cities to perform usability research with users</li>
         <li>recruit customers through
           <a href="https://www.gov.uk/government/news/help-improve-companies-house">our user panel</a>
           to take part in sessions at our usability lab in Cardiff</li>
         <li>build and test our services to meet
           <a href="https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction#meeting-government-accessibility-requirements">government accessibility requirements</a>
         </li>
-        <li>are developing
-          <a href="https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/">an assistive technology lab</a>
-          in our Cardiff office to improve the testing of our services and raise awareness of accessibility</li>
       </ul>
       <p>
         <a href="https://companieshouse.blog.gov.uk/category/user-research/">Read our blogs about usability</a>
         to find out more about our user research work.</p>
-      <p>This statement was prepared on 22 April 2021. It was last updated on 20 January 2022.</p>
+      <p>This statement was prepared on 22 April 2021. It was last updated on 8 October 2024.</p>
```

Comment:
8th October? Should this be November or does it not matter?

Reply:
Based on the doc they provided, it doesn't have to be accurate:
`Note for devs: this date does not need to be super accurate but should be close to when the page is republished.`
at the time of the update I had no idea when it will be published.